Jenkinsfile

```diff
@@ -43,57 +45,72 @@ pipeline {
                 }
             }
         }
+
         stage('Download') {
             steps {
-                script {
-                    def run_py_dl = sh(
-                        script: 'cd config;. venv/bin/activate; python3.7 run.py download', returnStatus: true
-                    )
-                    withCredentials([file(credentialsId: 's3cmd_kg_hub_push_configuration', variable: 'S3CMD_JSON')]) {
-                        if (run_py_dl == 0) { // upload raw to s3
-                            sh 'cd config; s3cmd -c $S3CMD_JSON --acl-public --mime-type=plain/text --cf-invalidate put -r data/raw s3://kg-hub-public-data/'
-                        } else { // 'run.py download' failed - let's try to download last good copy of raw/ from s3 to data/
-                            sh 'cd config; rm -fr data/raw; mkdir -p data/raw'
-                            sh 'cd config; s3cmd -c $S3CMD_JSON --acl-public --mime-type=plain/text --cf-invalidate get -r s3://kg-hub-public-data/raw/ data/raw/'
+                dir('./gitrepo') {
+                    script {
+                        def run_py_dl = sh(
+                            script: '. venv/bin/activate; python3.7 run.py download', returnStatus: true
+                        )
+                        withCredentials([file(credentialsId: 's3cmd_kg_hub_push_configuration', variable: 'S3CMD_JSON')]) {
+                            if (run_py_dl == 0) { // upload raw to s3
+                                sh 's3cmd -c $S3CMD_JSON --acl-public --mime-type=plain/text --cf-invalidate put -r data/raw s3://kg-hub-public-data/'
+                            } else { // 'run.py download' failed - let's try to download last good copy of raw/ from s3 to data/
+                                sh 'rm -fr data/raw; mkdir -p data/raw'
+                                sh 's3cmd -c $S3CMD_JSON --acl-public --mime-type=plain/text --cf-invalidate get -r s3://kg-hub-public-data/raw/ data/raw/'
+                            }
                         }
                     }
                 }
             }
         }
+
         stage('Transform') {
             steps {
-                sh 'cd config;. venv/bin/activate; python3.7 run.py transform'
+                dir('./gitrepo') {
+                    sh '. venv/bin/activate; python3.7 run.py transform'
+                }
             }
         }
+
         stage('Load') {
             steps {
-                sh 'cd config;. venv/bin/activate; python3.7 run.py load'
-                sh 'cd config;. venv/bin/activate; pigz merged-kg.tar'
+                dir('./gitrepo') {
+                    sh '. venv/bin/activate; python3.7 run.py load'
+                    sh '. venv/bin/activate; pigz merged-kg.tar'
```

Comment:
Why does this need activate?

Reply:
`run.py load` needs the venv environment - I guess `pigz` doesn't though